lib/browser/objects-registry.js

```diff
@@ -20,6 +20,7 @@ class ObjectsRegistry {
   add(webContents, obj) {
     // Get or assign an ID to the object.
     let id = this.saveToStorage(obj);
+    this.storage[id].count++;
```

Comment:
`remove` should never be called when there is still reference to the object, so if `setDestructor` is working correctly, this case should not happen.

> remove(webContents1, 1) // dev tools are open, don't need reference any more

1 is the id of `browserWindow `, so as long as you are able to change its size, you are holding a reference to it.